### DISCUSSION TITLE
Conda update HTTP 0000 Connection Failed

### DISCUSSION THREAD

**Discussion by AnthonyB08** [idaholab/moose#16271] [Q&A General] [answered]
Hi, 

Last week I had a similar problem (see png) and had solved it by .. just re-typing conda update --all ..and it eventually worked. 
Now it is doing it again, but my receding method is not working. What do you guys suggest? 

![issue](https://user-images.githubusercontent.com/55299169/99862432-bac48800-2b67-11eb-81c7-7d37b5ab30b2.PNG)

**Comment by dschwen** [comment]
What is your OS, VM yes or no, how did you install conda, do you do any other python development on that machine? Your python installation (requests library) could be broken. Can you access the network at all?

**Reply by AnthonyB08** [reply]
Re: Just tried the update command and it had worked. Interesting.

**Comment by AnthonyB08** [comment]
I am running Windows, WSL2, with Unbuntu 20.04 LTS. No, I do not do any
other python development. I am apart of University Massachusettes Lowell
Linux Club where a couple of Nuclear students and faculty are trying to
learn the MOOSE framework. I was able to access the network on November
13th. Additionally, the group and I had a *channel initialization *error
while running example 1.

**Comment by AnthonyB08** [comment, marked as answer]
RE:

On November 13th, I had successfully update conda by continuously typing *conda
update --all* until it finally worked (after ~ 10 tries)

**Comment by milljm** [comment]
This is an [Anaconda repository](https://repo.anaconda.com/pkgs/) issue. Sometimes there are hiccups when dealing with that service. 'Trying again' is likely the only thing one can do...
